packages/web/app/src/components/organization/members/list.tsx

```diff
@@ -1,5 +1,6 @@
-import { useCallback, useMemo, useState } from 'react';
-import { MoreHorizontalIcon, MoveDownIcon, MoveUpIcon } from 'lucide-react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
We have a debounce helper you can use:
`import { useDebouncedCallback } from 'use-debounce';`